addons/microsoft_calendar/data/neutralize.sql

```diff
@@ -0,0 +1,7 @@
+-- disable microsoft calendar integration
+UPDATE res_config_settings
+   SET cal_microsoft_client_id = NULL,
```

Comment:
this is an example of a neutralization not happening: the intended effect is to update `ir_config_parameter` because that's what the corresponding config field is doing.